packages/terra-functional-testing/src/commands/utils/getViewports.js

```diff
@@ -0,0 +1,16 @@
+const { TERRA_VIEWPORTS } = require('../../constants');
+
+const ALL_VIEWPORTS = Object.keys(TERRA_VIEWPORTS);
+
+/**
+ * Returns an array of viewport dimensions.
+ * @param {string|string[]} sizes - A list of Terra viewport sizes.
+ * @return {array} - An array of viewports. Returns all viewports if sizes are not provided.
+ */
+const getViewports = (...sizes) => {
+  const viewports = sizes.length > 0 ? sizes : ALL_VIEWPORTS;
+
+  return viewports.map((name) => TERRA_VIEWPORTS[name]);
```

Comment:
I kept this the same, but it might make sense to have this return the name too depending on how people are consuming this. It might look something like this:

```js
  return viewports.map((name) => {
    const { height, width } = TERRA_VIEWPORTS[name];

    return { height, width, name }; // or { ...TERRA_VIEWPORTS[name], name }
  });
```

Reply:
Do we even need this helper? Could we just expose the 'TERRA_VIEWPORTS' map directly?

Reply:
Consumers could get all viewports by Object.values, and always know what name relates to what size.

Reply:
I was thinking something similar when I started making these changes. This is one of the topics that was brought up early in our WDIO sync and we settled on keeping this for passivity. Quite a few consumers are using this, but they should probably be using `describeViewports`

Reply:
If this is hanging out for passivity, i wouldn't update the api.